src/transform/ReferenceCollection.js

```diff
@@ -225,8 +266,23 @@ const collectNearbyReferences = (document, sourceNode) => {
   return new NearbyReferences(selectedIndex, referencesGroup)
 }
 
+/**
+ * Collect nearby references.
+ * @param {!Document} document
+ * @param {!Node} sourceNode
+ * @return {!NearbyReferencesV2}
+ */
+const collectNearbyReferencesV2 = (document, sourceNode) => {
```

Comment:
May be nice to have a more declarative name (`V2` isn't very informative 😂) maybe `collectNearbyReferencesAsText`?

Reply:
Sounds good! Done

Reply:
I'm still seeing `V2`... is there a way you could describe what's different about this new class and name it accordingly?

Reply:
Changed from `ReferenceItemV2` to `ReferenceLinkItem` since it contains the `href` and `text` inside a `<a>` tag. Not sure if it works for you as well?